src/multiplayer-servers/multiplayer-services/server-protection/protocols.md

```diff
@@ -0,0 +1,68 @@
+# Ports and Protocols
+
+Game server ports are often vulnerable to attacks. 
+GameFabric offers Protocols to protect your game server traffic with powerful SteelShield mitigations.
+
+## Pre-requisites
+
+In order to protect the your game server traffic, your GameFabric installation requires:
+
+- Protection support enabled, usually indicated by the presence of the Protection/SteelShield navigation item.
+- Sites that support SteelShield, e.g. bare-metal.
+
+## Mitigations
+
+SteelShield provides mitigations that are tailored to check and let through only specific game or network traffic.
+To make use of the pre-defined mitigations you must create a new Protocol.
+
+## Protocols
+
+A Protocol – sometimes referred to as Protection Protocol, to distinguish it from a 
+network protocol such as UDP or TCP –
+describes the application protocol used by your game traffic or service.
+
+When you create a new Protocol, you must select a Mitigation and a network protocol.
+The network protocol that you select must be compatible with the Mitigation and their network protocol in order to be selectable.
```

Comment:
"The network protocol must be compatible with another network protocol" That is confusing, it is explained 2 paragraphs later but I would suggest changing it here:

```
Some Mitigations only support specific network protocols, such as UDP and TCP.
In those cases, it is impossible to select a network protocol that is not supported by the Mitigation you selected.
```

WDYT? IMO clears up the topic and even makes the other paragraphs redundant.

Reply:
Sounds good, applied.